src/main/java/ca/on/oicr/pineryreports/reports/impl/PreciseIdentityReport.java

```diff
@@ -0,0 +1,233 @@
+package ca.on.oicr.pineryreports.reports.impl;
+
+import static ca.on.oicr.pineryreports.util.GeneralUtils.DATE_REGEX;
+import static ca.on.oicr.pineryreports.util.GeneralUtils.REPORT_CATEGORY_COUNTS;
+import static ca.on.oicr.pineryreports.util.SampleUtils.*;
+
+import ca.on.oicr.pinery.client.HttpResponseException;
+import ca.on.oicr.pinery.client.PineryClient;
+import ca.on.oicr.pinery.client.SampleClient;
+import ca.on.oicr.pineryreports.data.ColumnDefinition;
+import ca.on.oicr.pineryreports.reports.TableReport;
+import ca.on.oicr.pineryreports.util.CommonOptions;
+import ca.on.oicr.ws.dto.SampleDto;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import java.io.IOException;
+import java.util.*;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.ParseException;
+
+public class PreciseIdentityReport extends TableReport {
+  public static final String REPORT_NAME = "preciseidentity";
```

Comment:
Should be added to report option list in `Main#getMainOptions`

Reply:
Updated